docs/src/frontend/index.md

```diff
@@ -791,8 +791,8 @@ On networks that do not, this will always return zero.
 stdlib.transfer(from: Account, to: Account | Address, amount, token?: Token, opts?: TransferOpts) => Promise<void>
 ```
 
-Performs a transfer of `{!js} amount` from `{!js} from` to `{!js} to`,
-which are accounts, such as those returned by `{!js} connectAccount`.
+Performs a transfer of `{!js} amount` from `{!js} from`, which is an account, such as those returned by `{!js} connectAccount`;
+to `{!js} to`, which can be an account or an address. 
```

Comment:
I think this a bit verbose. How about something like: 

Performs a transfer of `amount` from `from` to `to`, where `from` is an account and `to` is either an account or address.